### ISSUE TITLE
Doesn't load specifically for embedded playlist URLs

### ISSUE THREAD

**Issue by Synetech** [ajayyy/SponsorBlock#1265]
The addon loads for most embedded playlist URLs except for specifically playlist URLs:

```
 ☐ https://www.youtube.com/embed/ playlist    ?list=PL0123456789ABCDEFGHIJKLMNOPQRSTUV
 ☑ https://www.youtube.com/embed/ aBCdEFgHIjK ?list=PL0123456789ABCDEFGHIJKLMNOPQRSTUV
 ☑ https://www.youtube.com/embed/ videoseries ?list=PL0123456789ABCDEFGHIJKLMNOPQRSTUV
 ☑ https://www.youtube.com/embed/             ?list=PL0123456789ABCDEFGHIJKLMNOPQRSTUV
```

**Comment by mchangrh** [comment]
real link for testing: `https://www.youtube.com/embed/playlist?list=PL8mG-RkN2uTx9Er8AqFDnD6mFvje5fybs`

This should've been fixed by #1116 but I have a feeling the URL is making it lookup the "playlist" videoID

**Comment by mchangrh** [comment]
.take
